src/components/Popover/src/PopoverBubble.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div
+		:class="$s.PopoverBubble"
+		:style="computedStyles"
+	>
+		<!-- @slot Popover bubble content -->
+		<slot />
+	</div>
+</template>
+
+<script>
+import chroma from 'chroma-js';
+
+export default {
+	name: 'PopoverBubble',
+
+	props: {
+		/**
+		 * Text color within the popover
+		 */
+		color: {
+			type: String,
+			default: '#000',
+			validator: (color) => chroma.valid(color),
+		},
+		/**
+		 * Background color of the popover
+		 */
+		bgColor: {
+			type: String,
+			default: '#fff',
+			validator: (color) => chroma.valid(color),
+		},
+	},
+
+	computed: {
+		computedStyles() {
+			return {
+				'--popover-color': this.color,
+				'--popover-bg-color': this.bgColor,
+			};
+		},
+	},
+};
+</script>
+
+<style module="$s">
+.PopoverBubble {
```

Comment:
I think we need to account for a default output that includes the desired padding in the container. Right now, the out-of-box solution requires additional styling. Rather than enforcing additional styling, let's default the padding and add support to override it. The default should have 24px padding:

![CleanShot 2021-11-09 at 15 18 56](https://user-images.githubusercontent.com/129122/141021022-2bb53f76-a7fb-4c40-9763-2e5386ad5b60.png)

But in scenarios like Menu you will need a prop to override that padding.

Reply:
Added!